go-controller/pkg/ovn/master.go

```diff
@@ -1030,6 +1030,19 @@ func (oc *Controller) addNode(node *kapi.Node) ([]*net.IPNet, error) {
 		return nil, err
 	}
 
+	// delete stale chassis in SBDB if system-id of the specific node has changed.
```

Comment:
@cathy-zhou could you pull this block out into a function called something like `deleteStaleNodeChassis(node)` and then also use it from `WatchNodes()`? It looks like the same code in both places.

Reply:
sure